src/dialogs/more-info/components/cover/ha-more-info-cover-tilt-position.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  TemplateResult,
+  unsafeCSS,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { styleMap } from "lit/directives/style-map";
+import { computeAttributeNameDisplay } from "../../../../common/entity/compute_attribute_display";
+import { stateColorCss } from "../../../../common/entity/state_color";
+import "../../../../components/ha-control-slider";
+import { CoverEntity } from "../../../../data/cover";
+import { UNAVAILABLE } from "../../../../data/entity";
+import { HomeAssistant } from "../../../../types";
+
+function createTiltSliderTrackBackgroundGradient(
+  count = 24,
+  minStrokeWidth = 0.2
+) {
+  const gradient: [number, string][] = [];
+
+  for (let i = 0; i < count; i++) {
+    const stopOffset1 = i / count;
+    const stopOffset2 =
+      stopOffset1 +
+      (i / count ** 2) * (1 - minStrokeWidth) +
+      minStrokeWidth / count;
+
+    if (i !== 0) {
+      gradient.push([stopOffset1, "transparent"]);
+    }
+    gradient.push([stopOffset1, "var(--control-slider-color)"]);
+    gradient.push([stopOffset2, "var(--control-slider-color)"]);
+    gradient.push([stopOffset2, "transparent"]);
+  }
+
+  return unsafeCSS(
```

Comment:
Is `unsafeCSS` needed here? As it is used in a style tag?

Reply:
No it's needed 😞 I got this error :
```
Argument of type 'string' is not assignable to parameter of type 'number | CSSResultGroup'.ts(2345)
```